README.adoc

```diff
@@ -48,11 +48,12 @@ Please, complement the content of this readme document with the link:https://git
 Make sure you have the following tools:
 
 * The link:http://golang.org/doc/install[Go Programming Language]
-** Currently, Kiali releases are built using Go 1.16. Although Kiali may build correctly using other versions of Go, it's suggested to use version 1.16.2 for development to ensure replicatable builds. Makefiles will require this minimum version of Go.
+** Currently, Kiali releases are built using a specific minimum version of Go as declared in the [Makefile](https://github.com/kiali/kiali/blob/master/Makefile#L31). Although Kiali may build correctly using other versions of Go, it's suggested to use the version that the Makefile uses for your own development to ensure replicatable builds.
```

Comment:
sorry - this doesn't render correctly. I guess this is not correct adoc link. Just correct so it shows up as a link with text of "Makefile":
![image](https://user-images.githubusercontent.com/2029470/160145325-f9a35556-e799-4a2d-bb0d-a59e0896008d.png)

Reply:
Woops, sorry I glided over the fact that it's an adoc not md haha, fixed the syntax!

Reply:
![image](https://user-images.githubusercontent.com/37652070/160146238-bdd0699d-bf5d-4e51-93c9-cf7b413d1169.png)
Renders correctly now!